bench/vector_tile.js

```diff
@@ -56,9 +56,9 @@ Tile.Feature.read = function (pbf, end) {
 };
 Tile.Feature._readField = function (tag, obj, pbf) {
     if (tag === 1) obj.id = pbf.readVarint();
-    else if (tag === 2) obj.tags = pbf.readPackedVarint();
+    else if (tag === 2) obj.tags = pbf.readPackedVarint(obj.tags);
```

Comment:
Hmm, what would be the incompatibility here? `obj.tags = ...` is a noop in this case because `obj.tags` already equals the array that is returned.